src/how_to/experiment.md

```diff
@@ -13,16 +12,21 @@ When doing an experiment, there are two roles:
 
+
+
 ## Process
 
-The process is as follows:
+If you are an experienced Rust contributor who would like to start an experiment in-tree, the process is as follows:
 
-* The **experimentor** should open a PR adding a new feature gate to the compiler, along with an associated tracking issue. They should include a short write-up documenting the motivation and outline of what they are trying to achieve. 
-* The **lang-team champion** can "second" the PR, starting an FCP. Once the FCP completes, the PR can land and implementation work begins (always gated under the new feature gate).
+* Write-up a description of the problem you are trying to solve and the general shape of the solution you want to work on. Discuss it on Zulip or elsewhere to find a **lang-team champion**:
+    * The champion is the connection to the lang-team. They can check in with you from time to time to see how the work is going and relay those updates to the lang-team (of course, you're always welcome to join meetings yourself too!). They can also help to discuss problems that arise.
+* Once you've found a champion, open a PR adding a new feature gate to the compile and create an associated tracking issue.
+    * The PR should include a write-up documenting the motivation and outline of what they are trying to achieve. 
+    * The feature gate should be marked as 'experimental', so that users get warnings if they try to use it. This flag has to stay until an RFC is accepted, even if the implementation is in good shape.
+* The lang-team champion will "second" the PR, starting an FCP. Once the FCP completes, the PR can land and implementation work begins (always gated under the new feature gate).
     * **Approving a new feature gate does not imply support for the feature.** It implies only that the lang team thinks it is worth doing the experiment to see what results.
-    * *Note to lang team members:* If you have concerns about the feasibility or wisdom of the feature, the right course of action is usually to allow experimentation to continue, but ensure that your concerns are noted on the tracking issue. This allows the experimentors to try and gather data and address your concern.
-
-**In order to be stabilized, a full RFC will be required.** The goal of the experimentation period is simply to gain experience and information so that a better RFC can be authored.
+    * Note to lang team members: If you have concerns about the feasibility or wisdom of the feature, the right course of action is usually to allow experimentation to continue, but ensure that your concerns are noted on the tracking issue. This allows the experimentors to try and gather data and address your concern.
+* When you feel the design is ready, you write an RFC as normal with your proposal. The goal of the experimentation period is simply to gain experience and information so that a better RFC can be authored.
 
 [members]: https://www.rust-lang.org/governance/teams/lang
```

Comment:
I'd also suggest maybe something like:

### I have a competing proposal to something that's an existing experimental feature. Can I land my own experimental feature?

Yes, in general! You should follow the same process as an entirely different experimental feature would (lang team champion etc), and likely should work closely with the existing owners to collaborate on findings, but it's the explicit intent that during the experimentation phase we're willing to temporarily have duplication of sorts while we find the best path forward.

Reply:
Done!